### ISSUE TITLE
Fullscreen Not working in Linux

### ISSUE THREAD

**Issue by medemi68** [atom/electron#3711]
Hi, 

It appears that the Fullscreen option in the configuration is no longer working on Linux. It was working until I updated to the latest build (electron v0.35.4). (Couldn't tell you what the previous build is though.

It looks like it tries to go fullscreen (It flickers white a bunch of times) but ends up reverting back to 800x600 window size.

If you need more information please let me know. 

Miguel

**Comment by KeitIG** [comment]
Working fine with Ubuntu 14.04

in the console:
``` javascript 
require('remote').getCurrentWindow().setFullScreen(true);
```

**Comment by zcbenz** [comment]
Which operating system and desktop environment are you using?

**Comment by medemi68** [comment]
HI, I'm using Debian, and I'm not using a desktop environment at all. Just plain X11.

**Comment by zcbenz** [comment]
@medemi68 Can you try whether the fullscreen mode of Chromium browser 47 works for you? Electron basically uses the same code with Chromium.

Also does the `setFullScreen` API work? Or it is just the `fullscreen` option not working?

**Comment by netlovers** [comment]
I had the exact same issue a few days ago. When I set the window size in my main js file, it almost works (there is a 1px black border at the right and bottom side). The same thing happens if I set the width and height to very big values. But F11 or setting fullscreen in any way didn't help.

One thing that helps is using a very minimal window manager like `matchbox` or `ratpoison`, but there is still a problem that these can't really handle alerts and confirm dialogs.
